sdk/examples/wallet/09_mint_native_token.rs

```diff
@@ -47,21 +52,23 @@ async fn main() -> Result<()> {
         foundry_metadata: None,
     };
 
+
```

Comment:
Why? :trollface:

Reply:
For philosopical reasons ... the emptiness is something we can't evade!